docs/installation.md

```diff
@@ -1,110 +1,109 @@
 # Installation
 
-## Prerequisites
-
-scvi-tools can be installed via conda or pip. If you don't know which to choose, we recommend conda.
+## Quick install
 
-### conda
+scvi-tools can be installed via `conda` or `pip`. We recommend installing into a virtual environment
+to avoid conflicts with other packages.
 
-1. Install conda. We typically use the [mambaforge] distribution. Use python>=3.9, and consider using mamba instead of conda. Mamba is a drop-in replacement for conda that is significantly more efficient.
```

Comment:
can we keep this?

Reply:
I updated it to this:

> For `conda`, we recommend using the Miniforge distribution, which is generally faster than the
official distribution and comes with conda-forge as the default channel (where scvi-tools is
hosted).

Reply:
The repo recommends using miniforge over mambaforge now